iam/iam-policy-for-ecs/main.tf

```diff
@@ -0,0 +1,352 @@
+data "aws_iam_user" "main" {
+  user_name = var.username
+}
+
+locals {
+  staging_developer_access_projects    = try(data.aws_iam_user.main.tags.staging-developer-access-projects, "")
+  staging_admin_access_projects        = try(data.aws_iam_user.main.tags.staging-admin-access-projects, "")
+  production_developer_access_projects = try(data.aws_iam_user.main.tags.production-developer-access-projects, "")
+  production_admin_access_projects     = try(data.aws_iam_user.main.tags.production-admin-access-projects, "")
+
+  staging_read_access_projects = distinct(compact(concat(
+    split(":", local.staging_developer_access_projects),
+    split(":", local.staging_admin_access_projects),
+  )))
+
+  production_read_access_projects = distinct(compact(concat(
+    split(":", local.production_admin_access_projects),
+    split(":", local.production_developer_access_projects),
+  )))
+
+  staging_write_access_projects = distinct(compact(split(":", local.staging_admin_access_projects)))
+
+  production_write_access_projects = distinct(compact(split(":", local.production_admin_access_projects)))
+
+  read_access_projects = concat(
+    local.staging_read_access_projects,
+    local.production_read_access_projects
+  )
+
+  write_access_projects = concat(
+    local.staging_write_access_projects,
+    local.production_write_access_projects
+  )
+
+  region = "eu-central-1"
```

Comment:
Hardcoded this value for now but will think about how to manage this dynamically.
Perhaps we can take the assignment of policy to the user from the module.

Reply:
@marcqualie should we prepend the region to the policy names, that we need for each geo region?